spec/unit/provider/group/windows_adsi_spec.rb

```diff
@@ -268,4 +268,10 @@
   it "should prefer the domain component from the resolved SID" do
     expect(provider.members_to_s(['.\Administrators'])).to eq('BUILTIN\Administrators')
   end
+
+  it "should raise an error on a bad username" do
+    user_name = 'b_a_baracus'
```

Comment:
I'm not saying I have a user by that name, but you might insert some random characters so this doesn't fail in weird and unusual ways.